service-python/pdfgenerator/src/lib/settings.py

```diff
@@ -25,5 +25,5 @@
 codes = {
   "6602": "asthma",
   "7101": "hypertension",
-  "0000": "hypertensionv1"
+  "0000": "hypertensionv2"
```

Comment:
When I put in `0000` to mas endpoint it says out of scope. 7101 created the new sideways pdf.